video/api/Test/Tests.cs

```diff
@@ -96,7 +96,7 @@ void TestShotsGcs()
         void TestLabels()
         {
             ConsoleOutput output = _analyze.Run("labels",
-                DownloadGcsObject(@"gs://demomaker/cat.mp4"));
+                DownloadGcsObject(@"gs://cloud-samples-data/video/cat.mp4"));
```

Comment:
Can we make this a const so we update it only once in all tests in case it changes in the future?